docs/releases/1.8.0.md

```diff
@@ -0,0 +1,124 @@
+# Datafaker 1.8.0 (11 Feb 2023)
+
+The first release of 2023 of Datafaker has been created. After one year of Datafaker, this marks
+the release of the last versioon compatible with Java 1.8. After this, Datafaker 2.x will have Java 17
```

Comment:
typo in word `version`

Reply:
Good catch, my brain managed to read right by that 🤷‍♂️

Reply:
my MacBook keyboard is broken. Thanks!

Reply:
should be fixed now